Atomic/help.py

```diff
@@ -0,0 +1,41 @@
+from . import Atomic
+import subprocess
+from pydoc import pager
+from os import environ
+from . import mount
+import sys
+
+class AtomicHelp(Atomic):
+    def help(self):
+        """
+        Displays the image/containe help file in a pager
+        :return: None
+        """
+        # Set the pager to less -R
+        dm = mount.DockerMount("/tmp")
+        enc=sys.getdefaultencoding()
+        environ['PAGER'] = '/usr/bin/less -R'
+        docker_id = self.get_input_id(self.args.docker_object)
+        self.inspect = self._inspect_image(docker_id)
```

Comment:
I think we should just mount it and read the file if it exists rather then cloning and executing code within the container.